CMakeLists.txt

```diff
@@ -70,17 +72,28 @@ file(GLOB TINYGETTEXT_HEADERS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} include/tinyg
 
 add_library(tinygettext ${TINYGETTEXT_SOURCES})
 
+## Fix Windows exports
+
+INCLUDE (GenerateExportHeader)
+GENERATE_EXPORT_HEADER( tinygettext
+             BASE_NAME tinygettext
+             EXPORT_MACRO_NAME TINYGETTEXT_API
+             EXPORT_FILE_NAME tinygettext_Export.h
+             STATIC_DEFINE tinygettext_BUILT_AS_STATIC
+)
+
+
 ## Add tinygettext dir to search path
-target_include_directories(tinygettext PUBLIC include/)
+target_include_directories(tinygettext PUBLIC include/ ${CMAKE_CURRENT_BINARY_DIR})
 
 ## SDL/iconv usage
 option(HAVE_SDL "Use SDL_iconv instead of system iconv" FALSE)
 if(HAVE_SDL)
 set_source_files_properties(${TINYGETTEXT_SOURCES} PROPERTIES COMPILE_DEFINITIONS HAVE_SDL)
-TARGET_INCLUDE_DIRECTORIES(tinygettext PUBLIC ${SDL2_INCLUDE_DIRS})
+INCLUDE_DIRECTORIES(tinygettext PUBLIC ${SDL2_INCLUDE_DIRS})
 TARGET_LINK_LIBRARIES(tinygettext PUBLIC ${SDL2_LIBRARIES})
 
-else(HAVE_SDL)
+endif(HAVE_SDL)
```

Comment:
The whole point of this is to link to SDL *or* iconv; tinygettext does not require both. So why is this here?

And what is wrong with TARGET_INCLUDE_DIRECTORIES? You left it alone in [line 87](https://github.com/SuperTux/tinygettext/pull/1/files#diff-af3b638bc2a3e6c650974192a53c7291R87).